math/src/elliptic_curve/short_weierstrass/curves/bls12_381/twist.rs

```diff
@@ -154,4 +205,16 @@ mod tests {
         let expected = BLS12381TwistCurve::create_point_from_affine(expectedx, expectedy).unwrap();
         assert_eq!(p.operate_with(&q), expected);
     }
+
+    #[test]
+    fn check_generator_g2_in_subgroup() {
+        let gen = BLS12381TwistCurve::generator();
+        assert!(gen.is_in_subgroup());
+    }
+
+    #[test]
+    fn check_arbitrary_g2_point_in_subgroup() {
+        let arb_point = BLS12381TwistCurve::generator().operate_with_self(420_u32);
+        assert!(arb_point.is_in_subgroup());
+    }
```

Comment:
Please add a test that checks that `is_in_subgroup` returns false for a point not in $G_2$.